packages/types/lib/deploy/api.ts

```diff
@@ -1,15 +1,15 @@
 import type { JSONSchema7 } from 'json-schema';
 
 import type { Endpoint, ApiError } from '../api.js';
-import type { IncomingFlowConfig, PostConnectionScriptByProvider } from './incomingFlow.js';
+import type { IncomingFlowConfig, OnEventScriptsByProvider } from './incomingFlow.js';
 import type { SyncDeploymentResult } from './index.js';
 
 export type PostDeployConfirmation = Endpoint<{
     Method: 'POST';
     Path: '/sync/deploy/confirmation';
     Body: {
         flowConfigs: IncomingFlowConfig[];
-        postConnectionScriptsByProvider: PostConnectionScriptByProvider[];
+        onEventScriptsByProvider: OnEventScriptsByProvider[];
```

Comment:
nothing to change: What you did is smart, but a bit bothered that it's not representative of the truth